src/editor-sidebar/save-panel.js

```diff
@@ -96,6 +97,22 @@ export const SaveThemePanel = () => {
 						} );
 					} }
 				/>
+				<CheckboxControl
+					label="Process Only Saved Templates"
```

Comment:
We're using the word "save/saved" a lot in this panel, so I'm wondering if swapping "saved" for "modified" here would make things more clear.

```suggestion
					label="Process Only Modified Templates"
```

Reply:
Yes, I agree.  Good call.